src/components/dev-hub/academia-sign-up-form.js

```diff
@@ -0,0 +1,213 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { colorMap, size, screenSize } from './theme';
+import { P, ArticleH3, H3 } from './text';
+import Input from './input';
+import Button from './button';
+import FormSelect from './select';
+import Checkbox from '@leafygreen-ui/checkbox';
+import SectionHeader from './section-header';
+import SuccessIcon from './icons/success';
+
+const StyledButton = styled(Button)`
+    display: flex;
+    margin: ${size.large} auto;
+`;
+
+const StyledInput = styled(Input)`
+    height: 40px;
+    margin-bottom: ${size.large};
+    width: 335px;
+`;
+
+const StyledFormSelect = styled(FormSelect)`
+    color: ${colorMap.greyLightTwo};
+    height: 40px;
```

Comment:
This actually breaks the dropdown, since we have to do some absolute positioning to get the effects we want. For the inputs can we just use `narrow` (even if it is a bit larger than the designs)? We can update the form elements at another time if this is what Dan really wants, but for now let's stick with `narrow` for all of them (aka remove `height: 40px` from styled form elements and just pass `narrow` as a prop).

Reply:
This also gives some wonky effects when using the different form elements because they have some complicated css going on to get the gradient border and shifting up text effects.